### ISSUE TITLE
Tutorial on Sampling

### ISSUE THREAD

**Issue by Razcle** [blei-lab/edward#854]
I've noticed that there are a lot more tutorials/examples around VI
but less around sampling. I thought I might write up some of the sampling 
examples as notebooks and add a sampling example for bayesian nets.

Is this something that would be helpful?

**Comment by dustinvtran** [comment]
Absolutely. Do you mean something specific by Bayes nets? One suggestion is to write a notebook on something sampling is the de facto approach for----such as the item response theory model or Rasch model.

**Comment by Razcle** [comment]
Sorry I realise the language was a little vague. I wanted both to just show how you can use Edward for sampling (where the Rasch model would be great) but also to demonstrate the benefits also of auto-diff by looking at applying HMC and SGHMC to sampling from the posterior of a neural network.

I've got a working version done this afternoon but trying to find a good way to tune the learning rate and step size (otherwise the performance is a little disappointing).

At the moment I'm implementing things like trace-plots and convergence criteria by hand. Are their plans to include more diagnostics into Edward? If so I can open another issue as I'd happily implement them.

Thanks
